src/nixpacks/builder/docker.rs

```diff
@@ -317,6 +317,10 @@ impl DockerBuilder {
 
           WORKDIR {app_dir}
 
+          # Update channels
```

Comment:
I don’t think we should be referencing a third party channel for all builds, especially when the channel is outside of the main nixpacks repo. 

I also think the problem only existed for pnpm and npm/yarn worked as expected.

if possible I think we should fix this issue upstream vs installing node from a new channel.

Reply:
Alternatively we could use an overlay, which nixpacks already has support for. Similar to the rust provider https://github.com/oxalica/rust-overlay

Reply:
> I don’t think we should be referencing a third party channel for all builds

I will be hosted by you. Furthermore, as you can see inside the channel, these new packages only downloads the package manager from NPM, unzips it and binds it.

I also thought of putting it inside the docker image, but it wouldn't be updated as often